SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
```

Comment:
Any thinking how long are we going to backport security fixes ? Or we just advice users to upgrade to latest versions ?

Reply:
Yep, for sure.